api/db/changelog/db.changelog-261-rename-is-active-column.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We don't need a difference changelog, looks the column is just created in changelog 260.  can we just change column name there.

Reply:
I know, just didn't want to mess liquibase for you if you're testing things. I'll do that.